### ISSUE TITLE
Validation and accessibility

### ISSUE THREAD

**Issue by inastefansdottir** [inastefansdottir/Empty-repo-for-project-exam-board#18]
Use tools to check HTML/CSS validity and ensure the site meets accessibility standards.

- [x] Run WAVE to check accessibility (color contrast, labels, alt tags, aria labels)
- [x] Run lighthouse (performance, SEO, accessibility)
- [x] Use W3C HTML Validator to check for HTML issues
- [x] Fix any broken ARIA roles or labels 
- [x] Make sure to clear all console logs

**Comment by inastefansdottir** [comment]
![Image](https://github.com/user-attachments/assets/cc4969fd-a3e2-4cf0-9868-b8a3f632ad51)

There were some missing labels and empty buttons on the website I was able to fix that with aria-labels. The pink and light beige colors together triggers the contrast error but if I make the pink any darker or deeper it wont fit the brand/design so I kept it that way.

![Image](https://github.com/user-attachments/assets/13072bd0-35e4-403d-80a1-7d3ffefe61cd)

The performance test wasn't great because of chrome extensions, it was significantly better when I tested it in incognito mode:

![Image](https://github.com/user-attachments/assets/6ef703b6-ba17-44f9-9c6b-fe6a45d2416c)

**Comment by inastefansdottir** [comment]
I forgot that javascript links with a "type" is automatically deferred so I quickly fixed that and also the span inside the ul

![Image](https://github.com/user-attachments/assets/c752aa31-661d-4892-ac77-fa273ab47c1b)

**Comment by inastefansdottir** [comment]
I finished adding aria-labels for better accessibility and cleared all the console logs.
